Assets/Scripts/Throw/NextColor.cs

```diff
@@ -63,4 +72,20 @@ private ColorSO GetRandomColorSO()
     {
         return _gameManager.GetRandomColorSO();
     }
+
+    private ColorSO ThreeColors()
```

Comment:
May want to rename this to `ChooseNextColor`, since we can have any number of colors and randomness can be toggled.